store/src/sql/entities/event.rs

```diff
@@ -39,7 +39,7 @@ pub async fn rebuild_car(blocks: Vec<BlockRow>) -> Result<Option<Vec<u8>>> {
     Ok(Some(car))
 }
 
-#[derive(Debug, Clone)]
```

Comment:
this is the core change.  This is to prep for the next PR, which will replace the `blocks` field of this struct with the parsed `unvalidated::Event<Ipld>` instead, which doesn't support Cloning.

Reply:
The `unvalidated::Event` type can't be Clone right now because it uses a Box pointer internally.  We could change that to an Rc or do something else to make it cloneable.  But I kind of feel like having Events not be cloneable will force us to be smarter about moving things and not making extraneous copies.  Events could be large and cloning them a bunch seems pretty inefficient unless we really have to.

Reply:
Yeah, I agree with that. It can be convenient in tests but using an arc/box/rc is fine. I was also thinking about the "common" things to define on public types e.g. PartialEq, Eq, Hash. IPLD derives Eq because it doesn't really support floats which is mostly where that comes in. We don't have to add them now (or ever). A while back I wanted them on a couple tests and ended up just encoding things to bytes to compare, which is fine.